pages/post/index.tsx

```diff
@@ -74,6 +75,10 @@ export default function NewPost() {
   const pathname = usePathname();
   const savedPathname = useRef(pathname);
 
+
+  const [posts, setPosts] = useDraftStore([{ id: 1, content: '', threads: [] }]);
```

Comment:
this is not how the draft store works. you can see in other files, eg `pages/feeds/index.tsx` or same file in L69 how to use it. source file is `src/stores/useDraftStore.ts`